contact.html

```diff
@@ -0,0 +1,165 @@
+                         <span class="navbar-toggler-icon"></span>
+                    </button>
+ 
+                    <div class="collapse navbar-collapse justify-content-end fw-bolder" id="collapsibleNavbar">
+                        <ul class="navbar-nav">
+                            <li class="nav-item">
+                                <a class="nav-link" href="./index.html">Home</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">About</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="./services.html">Services</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Blog</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="./contact.html">Contact</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-12 col-sm-6 order-sm-1 mt-2"> 
+                    <a class="navbar-brand title fst-italic fs-6 py-3" href="#">Maria Hope, Ph.D. -Psychologist</a>
+                </div>
+
+            </nav>
+        </div>
+    </header>
+    
+    <!-- Main -->
+    <main>
+        <section class="container-fluid bg-limegreen p-5">
+            <div class="row justify-content-center my-3 py-5 ">
+                <div class="col-10 text-center">
+                    <h6 class=" fst-italic fw-lighter pt-5">Contact Me</h6>
+                    <h1 class=" fw-lighter lh-base">I'm a title. ​Click here to edit me.</h1>
+                    <p class="pt-3">I'm a paragraph. Click here to add your own text and edit me. I’m a great place for you to tell a story and let your users know a little more about you.</p>
+                    
+                </div>
+            </div>
+            <div class="row justify-content-center">
+                <div class="col-md-3  mb-2 ml-3">
+                    <input type="text" class="form-control" placeholder="Name" id="name">
+                </div>
+                <div class="col-md-3 mb-2">
+                    <input type="email" class="form-control" placeholder="Email" id="email">
+                </div>
+                <div class="col-md-6 mb-2 mx-3">
+                    <input type="text" class="form-control" placeholder="Subject" id="usr">
+                </div>
+                <div class="col-md-6  mb-2 mx-3">
+                    <textarea class="form-control" rows="5" id="comment">Comment</textarea>
+                </div>
+                <div class="col-md-6 mb-2 mx-3">
+                    <button class="appointment-button bg-mediummint my-2 w-100">Send</button>
+                </div>
```

Comment:
Your form looks great - could yo try and follow the design a bit more precisely though (black box on hover and focus) instead of relying on Bootstrap? As a front-end dev, you would be expected to follow the design precisely 😉 
Try the pseudo element `:focus`